src/init.cpp

```diff
@@ -917,7 +917,7 @@ bool AppInitBasicSetup()
 #endif
 
     if (!SetupNetworking())
-        return InitError("Initializing networking failed");
+        return InitError(_("Initializing networking failed"));
```

Comment:
Tend to NACK. This will put the translated message into the debug log

Reply:
I guess the translation should be registered here only and then later translated in InitError with `_()`